src/pages/task/ActionButtons.tsx

```diff
@@ -37,19 +38,22 @@ import {
   UNSCHEDULE_TASK,
 } from "gql/mutations";
 import { useUpdateURLQueryParams } from "hooks";
+import { TaskStatus } from "types/task";
 import { PreviousCommits } from "./actionButtons/previousCommits/PreviousCommits";
 import { TaskNotificationModal } from "./actionButtons/TaskNotificationModal";
 
 interface Props {
   initialPriority?: number;
+  isDisplayTask: boolean;
   isExecutionTask: boolean;
   task: GetTaskQuery["task"];
 }
 
 export const ActionButtons: React.VFC<Props> = ({
   initialPriority = 1,
-  task,
+  isDisplayTask,
```

Comment:
If we are passing task in any way why do we need to calculate isDisplayTask independently and pass it in as well? It seems like it can be calculated below as well.

Reply:
It could be calculated here for sure — but I decided to lift up `isDisplayTask` into the `Task` component so it would only be computed once. That way we can make sure the same logic is being used wherever we determine if a task is a display task. Open to repeating that logic in `ActionButtons` and `TaskTabs` if you think it makes more sense though.

Reply:
Great call! I missed that it was also being passed to TaskTabs